src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -33,7 +36,24 @@ void MatterPumpConfigurationAndControlClusterServerAttributeChangedCallback(cons
 {
     emberAfDebugPrintln("PCC Server Cluster Attribute changed [EP:%d, ID:0x%x]", attributePath.mEndpointId,
                         (unsigned int) attributePath.mAttributeId);
-    // TODO
+
+    switch (attributePath.mAttributeId)
+    {
+    case ControlMode::Id: {
+        uint8_t value;
+        ControlMode::Get(attributePath.mEndpointId, &value);
+        EffectiveControlMode::Set(attributePath.mEndpointId, value);
```

Comment:
Please remove the `disabled: true` from the tests in `src/app/tests/suites/certification/Test_TC_PCC_2_3.yaml`, ` and `src/app/tests/suites/certification/Test_TC_PCC_2_3.yaml` that should now be passing.

Reply:
This will be done in PR #17189